game.py

```diff
@@ -17,19 +17,9 @@ def wins(user):
     return False
 
 def is_approved(yay, nay, total):
```

Comment:
we don't need this, yea?

Reply:
ah it's still passed in by https://github.com/gitnomicadmin/gitnomic/blob/cbd181f54fe12b45ccfd54d1ef42dd67f7bae1c8/game.py#L74

Reply:
maybe my next turn will be to add a linter?